content/docs/querying/functions.md

```diff
@@ -80,6 +86,55 @@ and value across all series in the input vector.
 `floor(v instant-vector)` rounds the sample values of all elements in `v` down
 to the nearest integer.
 
+## `histogram_quantile()`
+
+`histogram_quantile(φ float, b instant-vector)` calculates the φ-quantile (0 ≤ φ
```

Comment:
I agree. Greek letters are "cool", but scary for the uninitiated, and I think very few Prometheus users will know the meaning of φ.